openvdb_cmd/vdb_tool/CMakeLists.txt

```diff
@@ -0,0 +1,159 @@
+endif()
+
+if(USE_ABC)
+  add_compile_definitions("VDB_TOOL_USE_ABC")
+  find_package(Alembic CONFIG REQUIRED)
+  target_link_libraries(vdb_tool_common INTERFACE Alembic::Alembic)
+endif()
+
+if(WIN32 AND (USE_ALL OR (USE_ABC AND USE_EXR)))
+  message(WARNING
+    " The OpenEXR and Alembic VCPKG packages are using conflicting Imath versions.\n"
+    " Disable one, if you encounter unresolved external symbols")
+endif()
+
+# Compiler flags
+
+# GCC flags
+if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+  add_compile_options(-Wno-invalid-offsetof -pthread -lpthread)
+  add_compile_options("$<$<CONFIG:DEBUG>:-O1>")
+endif()
+
+
+# MSVC flags
+# Increase the number of sections that an object file can contain
+add_compile_options("$<$<COMPILE_LANG_AND_ID:CXX,MSVC>:/bigobj>")
+# Excludes APIs such as Cryptography, DDE, RPC, Shell, and Windows Sockets
+add_compile_definitions("$<$<CXX_COMPILER_ID:MSVC>:WIN32_LEAN_AND_MEAN>")
+# Disable non-secure CRT library function warnings
+# https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/
+#     compiler-warning-level-3-c4996?view=vs-2019#unsafe-crt-library-functions
+add_compile_definitions("$<$<CXX_COMPILER_ID:MSVC>:_CRT_SECURE_NO_WARNINGS>")
+# Disable POSIX function name warnings
+# https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/
+#     compiler-warning-level-3-c4996?view=vs-2019#posix-function-names
+add_compile_definitions("$<$<CXX_COMPILER_ID:MSVC>:_CRT_NONSTDC_NO_WARNINGS>")
+
+if(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
+  add_compile_options("$<$<CONFIG:RELEASE>:/Oi>")
+
+  message(STATUS "Suppressing some noisy MSVC CXX warnings.")
+endif()
+# Conversion from int64_t to long
+add_compile_options("$<$<COMPILE_LANG_AND_ID:CXX,MSVC>:/wd4244>")
+# It's not possible to use STL types in DLL interfaces in a portable and
+# reliable way so disable this warning
+add_compile_options("$<$<COMPILE_LANG_AND_ID:CXX,MSVC>:/wd4251>")
+# Conversion from size_t to uLong
+add_compile_options("$<$<COMPILE_LANG_AND_ID:CXX,MSVC>:/wd4267>")
+# Non dll-interface class used as base for dll-interface class
+#add_compile_options("$<$<COMPILE_LANG_AND_ID:CXX,MSVC>:/wd4275>")
+# Truncation from 'int' to 'bool'
+#add_compile_options("$<$<COMPILE_LANG_AND_ID:CXX,MSVC>:/wd4305>")
+
+find_package(OpenVDB)
+if(NOT OpenVDB_FOUND)
+  message(FATAL_ERROR
+  " Couldn't find OpenVDB\n"
+  " Set OPENVDB_CMAKE_PATH to <OpenVDB install path>/lib/cmake/OpenVDB")
+endif()
+
+target_include_directories(vdb_tool_common INTERFACE "${Boost_INCLUDE_DIRS}" "${OpenVDB_INCLUDE_DIRS}" "include")
+target_link_libraries(vdb_tool_common INTERFACE TBB::tbb OpenVDB::openvdb)
+
+# vdb_tool
+add_executable(vdb_tool src/main.cpp)
+target_include_directories(vdb_tool PRIVATE vdb_tool_common)
+target_link_libraries(vdb_tool PRIVATE vdb_tool_common)
+
+
+# unit test
+if(BUILD_TEST)
+  find_package(GTest CONFIG REQUIRED)
+
+  add_executable(test src/unittest.cpp)
+  target_include_directories(test PRIVATE vdb_tool_common)
```

Comment:
Same as above, replace `test` to either be `vdb_tool_test` or `vdb_tool_unittest`.